node_modules/oae-principals/tests/test-users.js

```diff
@@ -1949,7 +1949,7 @@ describe('Users', function() {
                                             // Create user with an empty displayName
                                             RestAPI.User.updateUser(updateUserRestContext, userObj.id, {'displayName': '\n'}, function(err) {
                                                 assert.ok(err);
-                                                assert.equal(err.code, 400);
+                                                assert.equal(err.code, 401);
```

Comment:
Why should this be a 401 now? Isn't it a user update missing a display name, which should still be 400?